chainer/functions/array/moveaxis.py

```diff
@@ -0,0 +1,114 @@
+import six
+
+from chainer import cuda
+from chainer import function_node
+from chainer.utils import type_check
+
+
+def _normalize_axis_tuple(axis, ndim, xp):
+    if xp.isscalar(axis):
+        axis = (axis,)
```

Comment:
The coverage misses this line.  The check is redundant after `Moveaxis.__init__`.